osu.Game/Screens/OnlinePlay/Multiplayer/Match/Playlist/MultiplayerPlaylist.cs

```diff
@@ -121,28 +110,29 @@ private void updateState()
 
                 firstPopulation = false;
             }
+
+            // As a small optimisation, only the ID is required to match the selected item.
+            PlaylistItem? selectedItem = client.Room == null ? null : new PlaylistItem(new APIBeatmap()) { ID = client.Room.Settings.PlaylistItemId };
+            queueList.SelectedItem.Value = selectedItem;
+            historyList.SelectedItem.Value = selectedItem;
         }
 
-        private void playlistItemAdded(MultiplayerPlaylistItem item) => Schedule(() => addItemToLists(item));
+        private void playlistItemAdded(MultiplayerPlaylistItem item) => Scheduler.Add(() => addItemToLists(item));
```

Comment:
The only functional difference between `Schedule()` + `Scheduler.Add()` is that the former respects transform delays. This is not desired in any case in multiplayer, `Scheduler.Add()` just makes this behaviour more explicit, and matches all other such implementations.